api/comics/management/commands/data.py

```diff
@@ -0,0 +1,4 @@
+descriptions = ['Gamer','Hercules and the Amazon Women','Pirates of the Caribbean: Dead Mans Chest','Moneyball','M for Mother (Mim mesle madar)','Julie','Loggerheads','Mr. Vampire (Geung si sin sang)','Hamlet, Prince of Denmark','The Lost Prince']
+publishers = ["Marvel", "DC", "Dark Horse", "Titan Comics"]  
+names = ["Daredevil", "X-men", "Spiderman"]
+covers = ["ddevil.jpg","afterlife.jpg", "arkham.jpg", "dead.jpg", "deadman.jpg","detective.jpg","fables.jpg","lantern.jpg","last.jpg","preacher.jpg","superman.jpg","wolverine"]
```

Comment:
We want URLs of comic covers in here, so don't have to add the images to the project

Reply:
don't know how to do this - the way the rest framework works it looks for images in the media folder, you can add local folders and tell it to save them in media/comics/whatever.

Reply:
Sorry, I meant to add a follow-up comment to this - if you go to models.py and change this line `cover = models.ImageField()` to this `cover = models.URLField()` then it will just store it as a URL, rather than as an image reference

Reply:
And then you can replace the "*.jpg" strings with a few image urls from around the web